src/dec/engines/SimpleDecodingEngine.ts

```diff
@@ -0,0 +1,72 @@
+import * as _ from 'lodash';
```

Comment:
What are the changes between this decoding engine and the one that was deleted?

Reply:
Just the addition of the `decodeLogs` function, and renaming the class from `SimpleTxDecodingEngine` to `SimpleDecodingEngine` as it now decodes both transactions and logs. Git picked it up as a file deletion + file addition in this case, but it's really a rename + addition of lines.